NATO Resupply tweak

loxley314 opened the pull request:
Changed on USAF Templeate:
bluRifle changed to Rifle without GL
Added suppressor category
Added M136 sight (M136 without sight is useless)
Added 40mm Shells for _m320 weapons
Nato Crate:
1 M136 sight for each tube
Added 40mm for the GLRifle
Added 40mm extra ammo from Tier 2
Uncommented supressor from Tier 2

loxley314 committed:
Resupply tweak

Changed on USAF Templeate:
bluRifle changed to Rifle without GL
Added suppressor category
Added M136 sight (M136 without sight is useless)
Added 40mm Shells for _m320 weapons
Nato Crate:
1 M136 sight for each tube
Added 40mm for the GLRifle
Added 40mm extra ammo from Tier 2
Uncommented supressor from Tier 2

StefArma commented:
Hi! 

About the GL, as you can see there are no HE ammo, the GL is intended to give player the ability to use flares and actually fight at night.

About suppressor can be a thing... wasn't sure exactly at which tier they should be good to have

loxley314 committed:
Update NATOCrate.sqf

Update to complain Stef request

loxley314 commented:
Hi, 
What do you think about it right now?.
I changed it again to give GL weapons instead normal one.
Removed GL Grenades from beginning and added a few of them from tier 2 and a bit more on tier 3.

About suppressor in original file was from BE_currentStage > 0, just was commented because it wasn't on template (on BLUE_NATO template was included).
Other option is give 1 - 2 suppressors on Tier 2 (just for the Team Leader and sharp shooter) and 5 suppressors on Tier 3 .

loxley314 committed:
Update BLUE_USAF.sqf

spelling correction

loxley314 commented on AntistasiOfficial.Altis/Templates/BLUE_USAF.sqf:
```diff
@@ -143,22 +148,28 @@
 	bluAttachments = [
 		"rhsusf_acc_anpeq15_light", //anpeq with laser + flashlight
-		"rhsusf_acc_nt4_black"		//Higer tier only
+		"rhsusf_acc_nt4_black"		//Higer tier only (suppressor category?)
```
Yep, was a copy paste from BLUE_NATO.sqf

zalexki commented:
This seems like a good simple working fix. 
@StefArma can you review ?

StefArma commented on AntistasiOfficial.Altis/Templates/BLUE_USAF.sqf:
```diff
@@ -71,9 +71,9 @@
-		"rhs_weap_hk416d10_m320",
-		"rhs_weap_m4_m320",
-		"rhs_weap_mk18_m320"
+		"rhs_weap_hk416d145",			// All changed to non-GL 
```
As i was saying, i'd prefer theese remain GL variants because it allow teamleader to use smoke and flare from the beginning.

loxley314 replied:
Yes, but the template should be only for weapon types. We can change on NATOcrate.sqf from blurifle to bluGL and then we got changed it also for servers that are not using RHS.

zalexki reviewed (approved):
looks good, not tested tho
